siliconcompiler/tools/__init__.py

```
@@ -56,6 +56,9 @@ def get_task(
                     continue
         tasks.add(task_obj)
 
+    if not tasks:
+        raise ValueError("No tasks found")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)